packages/cli/src/commands/build/index.ts

```diff
@@ -930,6 +931,20 @@ async function doBuild(
     throw existingConfig;
   }
   if (existingConfig) {
+    // Validate deploymentId if present (user-configured for skew protection)
+    if (
+      'deploymentId' in existingConfig &&
+      typeof existingConfig.deploymentId === 'string' &&
+      existingConfig.deploymentId.startsWith('dpl_')
+    ) {
+      throw new NowBuildError({
+        code: 'INVALID_DEPLOYMENT_ID',
+        message:
+          'The deploymentId cannot start with the "dpl_" prefix. Please choose a different deploymentId in your next.config.js.',
```

Comment:
I don't think we should say next.config.js here since the deployment id could come from anywhere, right?

The file on line 928 is config.json 🤔